lib/kennel/syncer.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 require "diff/lcs"
-require_relative "./output_limiter"
```

Comment:
Arbitrarily adding my comments here because line-of-code comments are the only way to thread in GitHub.

I really want to make Syncer smaller, not bigger. It's already large and complex and hard to test; let's not make that worse. Why do you want to get rid of this class?

Reply:
removing this class did not make syncer bigger
it still has the same logic as before "set max" + "build warning" + "call truncate"
(and now they are all in a single place, the `truncate_diff` method

Reply:
I'm not sure how it didn't make Syncer bigger. Syncer with the limiting built-in is more complex than Syncer without the limiting built in. Right?

Reply:
No wait I get it now, I misread something. Sorry.